.eslintrc.json

```diff
@@ -2,7 +2,7 @@
     "env": {
         "es2020": true,
         "node": true,
-        "mocha": true
+        "jest": true
```

Comment:
I would rather use a different test mechanism, such as https://github.com/lukeed/uvu or maybe https://github.com/davidmarkclements/brittle.
Jest has a ton of problems with ESM, injects stuff, and provides much more overhead that needed

Reply:
i would recommend uvu but keep in mind it has no reporting functionality for if we want better test results in PRs (i.e. a format github integrations can understand). either way we should probably tackle that in a separate pr since this is already a monolith

Reply:
Agreed that Jest isn't great with ESM. I generally like the batteries-included approach, especially when we want to tackle issues such as #211.

Reply:
yeah i agree. if we did use uvu, uvu and c8/istanbul usually works alright and we would have coverage integration. but we wouldn't have any useful test results output (and unfortunately the uvu author seems to be against introducing that)

Reply:
Chromium has native coverage testing, which a small tool like [c8](https://github.com/bcoe/c8#readme) can expose.
Tape, brittle, uvu all support coverage

Reply:
Let's follow James' suggestion to tackle this in a separate PR. c8 seems super interesting, curious to see how that would look in a proper project.